.github/workflows/scripts/constant.js

```diff
@@ -0,0 +1,33 @@
+let CONSTENT_VALUES = {
+    GLOBALS: {
+        LABELS: {
+            STALE: "stale",
+            AWAITINGRES: "stat:awaiting response",
+            BUG: 'type:bug',
+            BUG_INSTALL: 'type:build/install',
+            TYPE_SUPPORT: 'type:support',
+            TYPE_OTHER: 'type:others',
+            TYPE_DOCS_BUG: 'type:docs-bug',
+            TYPE_PERFORMANCE: 'type:performance'
+        },
+        STATE: {
+            CLOSED: "closed"
+        },
+        TENSORFLOW_CORE_REPO: 'https://github.com/tensorflow/tensorflow/pull',
+        PR_TRIGGER_REPO: 'testRep,keras'
+    },
+    MODULE: {
+        CSAT: {
+            YES:'Yes',
+            NO:'No',
+            BASE_URL: 'https://docs.google.com/forms/d/e/1FAIpQLSfaP12TRhd9xSxjXZjcZFNXPG' +
+        'k4kc1-qMdv3gc6bEP90vY1ew/viewform?',
+            MEDIA_PIPE_BASE_URL: 'https://docs.google.com/forms/d/e/1FAIpQLScOLT8zeBHummIZFnfr9wqvxYzWD1DAypyvNia5WVIWtFANYg/viewform?',
```

Comment:
Why is the MediaPipe feedback form added here?

Reply:
Hi @Neilblaze ,
I've added MEDIA_PIPE _BASE_URL to make workflow repo independent. So without doing any changes it will work on all the repos.

Reply:
I didn't get what you exactly mean by "_to make workflow repo independent_"? Adding to the same, MediaPipe is completely isolated from this repo, so how will this "_work on all the repos_"?

Would you mind explaining this?

Reply:
We have two feedback forms: 
1. Mediapipe
2. Other repositories(tensorflow, tfx, serving, tfjs etc.)

I am keeping Mediapipe feedback URL inside the other repositories so that I can make identical code that will work on other repositories as well as on mediapipe without doing the changes for each repository.

Reply:
Gotcha! Thanks for the clarity.